app/(admin)/admin/notifications/page.tsx

```diff
@@ -42,30 +47,87 @@ const AdminNotifications = (): JSX.Element => {
     await sendEmailNotifications({ content, groupUsers, subject });
   };
 
+  /**
+   * Function to handle radio selection logic.
+   * @param value - Value of the radio buttons.
+   */
+  const handleRadioChange = (value: string): void => {
+    if (value === 'all') {
+      getLeagueData();
+    }
+  };
+
   return (
-    <section data-testid="admin-notifications-content">
-      <Button
-        label="Email Testers"
-        type="button"
-        onClick={getLeagueData}
-        data-testid="email-testers"
-      />
-      <form onSubmit={handleSubmit}>
-        <Label htmlFor="subject">Subject:</Label>
-        <Input
-          type="text"
-          id="subject"
-          data-testid="subject-text"
-          onChange={(e) => setSubject(e.target.value)}
-        />
-        <Label htmlFor="content">Content:</Label>
-        <textarea
-          name="content"
-          id="content"
-          onChange={(e) => setContent(e.target.value)}
-          data-testid="content-text"
+    <section
+      className="flex flex-col space-y-6"
+      data-testid="admin-notifications-content"
+    >
+      <p>
```

Comment:
I feel like making this text `<p>` doesn't make sense from a syntaxical standpoint. Could this not be a `<span>`?

Reply:
I use `<p>` for fragments and sentences, and `<span>`s for just a few words, number, or when I need to isolate words within a `<p>` (or other text tag) for styling purposes.

As such, I feel a `<p>` tag is appropriate more appropriate here.